.github/workflows/smoke-e2e-ci-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Smoke E2E Tests
+on: [push, pull_request]
+jobs:
+  against-real-network-on-linux:
+    timeout-minutes: 30
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        node: [16, 18]
```

Comment:
- Changed e2e CI to run against 18 and 20 since 16 will be EOL in ~3 weeks as of now (Aug 18th, 2023)
- Created couple of issues:
  - https://github.com/o1-labs/zkapp-cli/issues/471
  - https://github.com/o1-labs/snarkyjs/issues/1079

Reply:
Actually reverted back to `[16, 18]` since more work need to be done for Node 20 support. `Node 20` will complain on `--experimental-wasm-threads` usage: `node: bad option: --experimental-wasm-threads`.

Reply:
Ok, back to [18, 20] after the fix for Node 20 support 😄